pandas/tests/frame/indexing/test_coercion.py

```diff
@@ -164,7 +164,6 @@ def test_12499():
     tm.assert_frame_equal(df, expected)
 
 
-@pytest.mark.xfail(reason="Too many columns cast to float64")
```

Comment:
So this essentially tests https://github.com/pandas-dev/pandas/issues/46896?

Reply:
Yep correct

Reply:
It's basically a duplicate of #20476